float64 current
float64 s_current
float64 voltage
float64 left_pos
float64 left_rps
float64 right_pos
float64 right_rps